templates/data/opensearch.html

```diff
@@ -1,345 +1,500 @@
+          </p>
         </div>
-      </div>
-      <hr class="p-rule" />
-      <div class="row">
-        <div class="col-3">
-          <h3 class="p-heading--5">Vector databases</h3>
+        <div class="col-6 col-medium-3">
+          <hr class="p-rule--highlight" />
+          <div class="p-card--overlay">
+            <div class="row--50-50">
+              <div class="col">
+                <h3 class="u-align-text--left u-no-margin--bottom">Managed</h3>
+              </div>
+              <div class="col">
+                <h3 class="u-align-text--right u-no-margin--bottom">From $3099</h3>
+              </div>
+            </div>
+            <p class="u-align-text--right">Per node, per year</p>
+            <p>
+              Get there faster. Let us deploy and operate your OpenSearch cluster, in your cloud tenancy or in your data centre.
+            </p>
+            <hr class="p-rule is-muted" />
+            <div class="p-section--shallow  u-no-margin--bottom">
+              <ul class="p-list--divided">
+                <li class="p-list__item is-ticked">Backed by an SLA</li>
+                <li class="p-list__item is-ticked">MSP and SOC2 certified service</li>
+                <li class="p-list__item is-ticked">Low cost of ownership</li>
+              </ul>
+              <p>
+                <a href="/data/opensearch/managed" aria-controls="data-spark-modal">Explore Managed OpenSearch Solutions&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
         </div>
-        <div class="col-6">
-          <p>Use OpenSearch to store and index vectors and metadata, unlocking the ability to use low-latency queries to discover assets by degree of similarity.</p>
+        <div class="col-6 col-medium-3">
+          <hr class="p-rule--highlight" />
+          <div class="p-card--overlay">
+            <div class="row--50-50">
+              <div class="col">
+                <h3 class="u-align-text--left u-no-margin--bottom">Self-managed</h3>
+              </div>
+              <div class="col">
+                <h3 class="u-align-text--right u-no-margin--bottom">From $500</h3>
+              </div>
+            </div>
+            <p class="u-align-text--right">Per node, per year</p>
+            <p>
+              24/7, weekday or advanced fire-fighting support for OpenSearch from Canonical for up to 10 years per release track.
+            </p>
+            <hr class="p-rule is-muted" />
+            <div class="p-section--shallow  u-no-margin--bottom">
+              <ul class="p-list--divided">
+                <li class="p-list__item is-ticked">BExpert troubleshooting on hand</li>
+                <li class="p-list__item is-ticked">Security and bug fixes</li>
+                <li class="p-list__item is-ticked">Full stack coverage</li>
+              </ul>
+              <br />
+              <br />
```

Comment:
```suggestion
```
Is there a better solution we can use here? [br elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) should be used to add breaks in text, not to manipulate spacing. Could you have a look at the vanilla docs?

Reply:
Adding a `p-section` on `p-card--overlay` div seems to work nicely:
![image](https://github.com/user-attachments/assets/510b6316-4fc6-4746-9ea2-d6fbfc6e1c08)